crates/driver/src/domain/competition/solution/fee.rs

```diff
@@ -204,14 +204,14 @@ impl Fulfillment {
 /// - test_adjust_quote_to_out_market_buy_order_limits
 /// - test_adjust_quote_to_in_market_sell_order_limits
 /// - test_adjust_quote_to_in_market_buy_order_limits
-fn adjust_quote_to_order_limits(
+pub fn adjust_quote_to_order_limits(
```

Comment:
nit: I think this function would benefit from taking a struct with named fields (we have done many mistakes with the ordering of many unnamed u256 params in the past.